bower.json

```diff
@@ -79,7 +79,8 @@
     "jsurl": "https://github.com/Sage/jsurl.git#v0.1.5",
     "moment-duration-format": "^1.3.0",
     "angular-websocket": "^2.0.0",
-    "ovh-angular-tail-logs": "^1.1.2"
+    "ovh-angular-tail-logs": "^1.1.2",
+    "angular-elastic": "^2.5.1"
```

Comment:
You should use http://master.ui-kit.ovh/#!/oui-angular/textarea for your text areas. Ask @tremblaymath about improving the textarea to be resizable.

Reply:
You can use ui-kit textarea. We will add it soon or you can create a pull-request, it's a guideline requirement.

Reply:
Have fixed this in the latest commit by using a function to calculate row length. Please check the latest commit